scattering/van_hove.py

```diff
@@ -318,13 +321,7 @@ def vhf_from_pvhf(trj, partial_dict, water=False):
         if isinstance(atom_pair, tuple) == False:
             raise ValueError("Dictionary key not valid. Must be a tuple.")
         for atom in atom_pair:
-            # checks if the atoms in tuple pair are atom types
```

Comment:
Could you also make sure to check that the elements in the tuple are type `str`?